packages/xstate-react/package.json

```diff
@@ -73,8 +74,8 @@
     "jsdom": "^14.0.0",
     "jsdom-global": "^3.0.2",
     "lerna-alias": "3.0.3-0",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "18.0.0-rc.0-next-811634762-20220110",
+    "react-dom": "18.0.0-rc.0-next-811634762-20220110",
```

Comment:
Before landing this we should set up tests to run against both React 17 and React 18. It's totally doable - I just didn't bother to do it right now.

This should also only land **after** React 18 gets finally released.

Reply:
OTOH - I don't think now that this would add much value here. Stuff that works with React 18 should just work with React 17. So it might be enough to just test with React 18. We can always also revisit adding extra tests for React 17 in the future.